simcon_project/users/admin.py

```diff
@@ -28,4 +29,11 @@ class CustomUserAdmin(UserAdmin):
     ordering = ('email', )
 
 
-admin.site.register(CustomUser, CustomUserAdmin)
\ No newline at end of file
+admin.site.register(CustomUser, CustomUserAdmin)
+
+models = apps.get_models()
+for model in models:
```

Comment:
Can we also do something like this in conversation_templates/admin.py?

Reply:
We created an admin.py in the nested simcon_project so we wouldn't have to do an admin.py for both apps. The way we have it set up in that file seems to work and we actually need to register the user models app there too, we didn't include those for some reason.

Reply:
Ahh I see that, that's a good idea